### ISSUE TITLE
Remember last directory used in open file dialog.

### ISSUE THREAD

**Issue by tavdog** [zaproxy/zaproxy#5945]
As is typical with a lot of Java applications when using an open file dialog box it always opens to a default location doesn't remember the last directory used. 

When opening files multiple times it is very cumbersome to have to navigate to a deep directory for a word list or other file.

It would be great if the open file dialog kept it state and remembered the last directory used.

**Comment by psiinon** [comment]
Agreed. Some open file dialogs actually do this, but not all - that annoys me too ;)
